File: src/Stunts.DynamicProxy/Stunt.DynamicFactory.cs
namespace Stunts
{
    partial class Stunt
    {
        static Stunt()
        {
            StuntFactory.Default = new DynamicStuntFactory();
            OnInitialized();
        }

        /// <summary>
        /// Invoked after the default <see cref="StuntFactory.Default"/> 
        /// is initialized.
        /// </summary>
        static partial void OnInitialized();
    }
}

File: src/Stunts.DynamicProxy/Stunt.DynamicFactory.vb
Namespace Global.Stunts
    Partial Friend Class Stunt
        Shared Sub New()
            StuntFactory.[Default] = New DynamicStuntFactory
            OnInitialized()
        End Sub

        ''' <summary>
        ''' Invoked after the default <see cref="StuntFactory.Default"/> 
        ''' is initialized.
        ''' </summary>
        Partial Private Shared Sub OnInitialized()

        End Sub
    End Class
End Namespace